htlcswitch/link.go

```diff
@@ -1939,14 +1941,13 @@ func (l *channelLink) HtlcSatifiesPolicy(payHash [32]byte,
 		return failure
 	}
 
-	// We want to avoid accepting an HTLC which will expire in the near
-	// future, so we'll reject an HTLC if its expiration time is too close
-	// to the current height.
-	timeDelta := policy.TimeLockDelta
-	if incomingTimeout-timeDelta <= heightNow {
+	// We want to avoid offering an HTLC which will expire in the near
+	// future, so we'll reject an HTLC if the outgoing expiration time is too
+	// close to the current height.
+	if outgoingTimeout-l.cfg.ExpiryGraceDelta <= heightNow {
```

Comment:
Should we make this a strict inequality? Otherwise, this is equivalent to:
```golang
outgoingTimeout < heightNow + l.cfg.ExpiryGraceDelta + 1
```

The previous constant in the switch was 2, and via the non-strict inequality was actually enforcing 3. Now that the constant in main is 3, this is actually now `4+broadcastDelta`. I assume the intent is to keep this as 3?

Alternatively, the constant in main could be set to 2 again and maintain the current grace delta w/o changing the code here.

Reply:
Oh, I think I just changed it from 2 to 3 because 3 felt like a safer buffer but didn't give much thought to it. I'm fine changing it back to 2. Slight preference for non-strict inequality.